tamr_unify_client/dataset/resource.py

```diff
@@ -64,20 +64,21 @@ def attributes(self):
         alias = self.api_path + "/attributes"
         return AttributeCollection(self.client, alias)
 
-    def _update_records(self, updates, **json_args):
+    def _update_records(self, updates, ignore_nan=False):
```

Comment:
Alternatively, we could leave `**json_args`, pop `ignore_nan` from the resulting dictionary, and allow the remainder to be passed to the standard library `json.dumps`, where an error will be raised for invalid kwargs.

Reply:
I think being aggressive about removing these kwargs is a good thing. We have reasonable confidence that that only kwarg being used is `ignore_nan`. Worst-case, we can do a patch release that does what you suggest^ (pop `ignore_nan`, pass the rest to `json.dumps`) if we find out there are use-cases that use other kwargs.